### PR TITLE
Add script block for survey reminder

### PR DESCRIPTION
**Pull request by axemonkey**
*  Add survey reminder text to Alternative Help page, as discussed with @teneightfive and @acjnash , text abridged from email from Alice Muskett (LAA)

[Pivotal story 87295520](https://www.pivotaltracker.com/story/show/87295520)

### CODE REVIEW

File: cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html [hunk 1]
```diff
@@ -103,6 +103,12 @@ <h1 class="u-pullLeft">Alternative help</h1>
 
   <dir-pagination-controls on-page-change="pageChanged(newPageNumber)"></dir-pagination-controls>
 
+  <call-script>
```
Comment (teneightfive) [inline cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html]: Does this not need a condition? I think it should only appear if they have failed assessment but in a scope category or in a category that is out of scope.

Comment (teneightfive) [inline cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html]: Although, saying that, can you get to this page without meeting the above somehow?

Comment (axemonkey) [inline cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html]: As far as I can tell, no.

Comment (axemonkey) [inline cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html]: Ah. Yes you can. If you are in scope and have assigned a valid user, you can skip straight to alt-help.

I'll add some conditions.

**Comment by axemonkey** [comment]
retest this please

**Comment by teneightfive** [comment]
Looks good

**Comment by teneightfive** [comment]
retest this please

**Comment by teneightfive** [comment]
retest this please
